tests/features/README.md

```
@@ -87,6 +87,43 @@ When running in local server mode, the tests will:
+2. Ensure a `HardwareProfile` exists in the tenant namespace (`X_TENANT`).
+3. Export its name and expected adapter resources (must match the profile's `defaultCount` / `maxCount` for CPU and memory):
+
+```bash
+export TEST_HARDWARE_PROFILE="your-profile-name"
+export TEST_HARDWARE_PROFILE_CPU_REQUEST="1"
+export TEST_HARDWARE_PROFILE_MEMORY_REQUEST="1Gi"
+export TEST_HARDWARE_PROFILE_CPU_LIMIT="2"
+export TEST_HARDWARE_PROFILE_MEMORY_LIMIT="2Gi"
+```
+
+4. Grant the FVT test runner **`get`/`list` on `jobs`** in the tenant namespace (to inspect the adapter container). The test process uses in-cluster config or `KUBECONFIG`; it does not read `HardwareProfile` CRs or cluster-scoped CRDs.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)